index.html

```diff
@@ -128,15 +128,15 @@ <h2>Tonic</h2>
             <div class="card-block" id="arti-2">
                 <h2>Multi-post Stories</h2>
                 <ul class="card-frame">
-                    <li class="no-count">Canopy</li>
-                    <li>Full Stack Dev</li>
+                    <li class="no-count">CANOPY</li>
+                    <li>Back End Dev</li>
                     <li>2015</li>
                 </ul>
-                <p>Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.</p>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
                 <ul class="card-tags">
                     <li>html</li>
+                    <li>CSS</li>
                     <li>Javascript</li>
-                    <li>css</li>
                 </ul>
                 <button class="card-button" type="submit">See Project</button>
       </div>
```

Comment:
- [ ] Good job so far @brainskev, I see you have used a separate div for each projects, You should wrap evey project in a single `section tag`. 
** something like **
```html
<section id="works-container">
        <div class="project-card">
             <!-- project detail -->
        </div>
        <div class="project-card">
             <!-- project detail -->
        </div>
        <div class="project-card">
             <!-- project detail -->
        </div>
</section>
```
- [ ] Once you add the container section, make sure to use `display: grid` for aligning the items.
- [Optional] Also it's great you used `id` and `class` make sure you use id once in a single page. I see you used 
```HTML  
<div class="card-block" id="arti-2">
```
multiple times. This is against the use of `id`.